pkg/knuu/knuu.go

```diff
@@ -236,3 +159,66 @@ func (k *Knuu) handleTimeout(ctx context.Context) error {
 
 	return nil
 }
+
+func loadEnvVariables() error {
+	if err := godotenv.Load(); err != nil && !os.IsNotExist(err) {
+		return ErrCannotLoadEnv.Wrap(err)
+	}
+	logrus.Info("The .env file does not exist, continuing without loading environment variables.")
+	return nil
```

Comment:
won't this always log that the env file doesn't exist even if it does? 
Shouldn't we update to:
```suggestion
	err := godotenv.Load()
	if err != nil && !os.IsNotExist(err) {
		return ErrCannotLoadEnv.Wrap(err)
	}
	if os.IsNotExist(err) {
	        logrus.Info("The .env file does not exist, continuing without loading environment variables.")
	}
	return nil
```

Reply:
Nice catch!